Cartfile

```diff
@@ -5,3 +5,4 @@ github "kif-framework/KIF"
 github "st3fan/SWXMLHash"
 github "rs/SDWebImage"
 github "norio-nomura/Base32"
+github "DaveWoodCom/XCGLogger" ~> 1.8
```

Comment:
Ah yes of course. So, we don't pin the Cartfile, but we do commit the Cartfile.resolved to the repo. I think that is good enough.